bower.json

```diff
@@ -11,6 +11,10 @@
   ],
   "dependencies": {
     "lodash": "~2.4.1",
-    "mojular": "matharden/mojular#internal"
+    "mojular": "matharden/mojular#internal2",
```

Comment:
Is this branch going to be moved somewhere less @matharden-centric? @teneightfive?